spec/pardot/objects/custom_fields_spec.rb

```diff
@@ -0,0 +1,58 @@
+require File.expand_path(File.dirname(__FILE__) + '/../../spec_helper')
+
+describe Pardot::Objects::CustomFields do
+  
+  before do
+    @client = create_client
+  end
+  
+  describe "query" do
+    
+    def sample_results
+      %(<?xml version="1.0" encoding="UTF-8"?>\n<rsp stat="ok" version="1.0">
+        <result>
+          <total_results>1</total_results>
+            <customField>
+              <created_at>2019-11-26 13:40:37</created_at>
+              <crm_id null="true" />
+              <field_id>CustomObject1574793618883</field_id>
+              <id>8932</id>
+              <is_record_multiple_responses>false</is_record_multiple_responses>
+              <is_use_values>false</is_use_values>
+              <name>Ω≈ç√∫˜µ≤≥÷</name>
+              <type>Text</type>
+              <type_id>1</type_id>
+              <updated_at>2019-11-26 13:40:37</updated_at>
+          </customField>
+        </result>
+      </rsp>)
+    end
+    
+    before do
+      @client = create_client
+    end
+    
+    it "should take in some arguments" do
```

Comment:
`it "takes in some arguments"`

Meh, the rest of the gem uses "should", so 🤷‍♂

Reply:
:( yeah, same as above